specs/~dynamic-names.yml

```
@@ -139,6 +139,20 @@ tests:
     partials: { node: '{{content}}<{{#nodes}}{{>*template}}{{/nodes}}>' }
     expected: 'X<Y<>>'
 
+  - name: Dynamic Names - Dobule Dereferencing
+    desc: Dynamic Names can't be dereferenced more than once.
+    data: { dynamic: 'test', 'test': 'content' }
+    template: '"{{>**dynamic}}"'
+    partials: { content: 'Hello, world!' }
+    expected: '""'
+
+  - name: Dynamic Names - Composed Dereferencing
+    desc: Dynamic Names cannot have composed dereferencing.
```

Since "composed dereferencing" is not defined separately anywhere (and doesn't need to), I suggest formulating this a bit differently:

```suggestion
    desc: Dotted Names are resolved entirely before dereferencing begins.
```

---

Agreed